### PR TITLE
Add form

### PR DESCRIPTION
**Pull request by jlvFlores**
### For this PR, I:
- Created a method called `current_user` that returns the first user
- Added three new forms, `(new post, create comment, create like)` that all update the views and database with the user obtained from the current user method as each post's, comment's and like's author.
## P.S.
The wireframes provided by Microverse did not state where the forms should go, so for this activity I located the new post form under `posts/new`, assuming that the project will later have a path for the **current user** to add a new post. I also located the comments and likes forms on each post's show page `(users/:user_id/posts/:post_id)`, as I felt it would be easier for future users to add those from the same page that the post is on.
### Thank you and happy reviewing 🙋‍♂️

### CODE REVIEW

File: app/views/posts/show.html.erb [hunk 1]
```diff
@@ -7,3 +7,16 @@
...
+    <%= form.label :text %>
+    <%= form.text_area :text %>
+
+    <%= form.submit "Add Comment", class: "form-btn" %>
+  <% end %>
+
+  <%= form_with model: Like.new, url: user_post_likes_path(@post.author, @post), local: true do |form| %>
+    <%= form.submit "Like", class: "form-btn" %>
+  <% end %>
+</div>
```
Comment (jerryowusu) [inline app/views/posts/show.html.erb]: Good job so far. Everything is looking fine. However, according to the project requirements, there should be a form in the `views/comments/new.html.erb` file which accepts input for creating comments which is not present at the moment. Kindly create the file (`views/comments/new.html.erb`) and add this input for creating comments

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @jlvFlores , 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:

almost-paul-walker

## Highlights
Nice and well organized code :+1: 
Neat design :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/show.html.erb (hunk 1 above)
Comment (jlvFlores) [reply app/views/posts/show.html.erb]: Hello @jerryowusu could you elaborate a bit more. I've looked over the requirements and I don't see any requirements stating that the comments form must be created with a separate view.


image

**Review by TadesseDev — APPROVED** [review]
### Hi @jlvFlores 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
AmazingCreepyFrogmouth-max-1mb




## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**